opensrp-reveal/src/main/java/org/smartregister/reveal/util/RevealSyncConfiguration.java

```diff
@@ -43,7 +43,7 @@ public int getUniqueIdInitialBatchSize() {
     }
 
     @Override
-    public boolean isSyncSettings() {
-        return false;
+    public boolean disableSyncToServerIfUserIsDisabled() {
```

Comment:
Where is this used?

Reply:
`org.smartregister.sync.intent.SyncIntentService#doSync`